### ISSUE TITLE
Enable editor step navigation shortcuts for signed-out users

### ISSUE THREAD

**Issue by lasjorg** [freeCodeCamp/freeCodeCamp#55650]
### Describe the Issue

It would be nice if you didn't have to be signed in to use the n and p step navigation.

### Affected Page

https://www.freecodecamp.org/learn

### Your code

```



```

### Expected behavior

Step navigation without being signed in.

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by naomi-lgbt** [comment]
Yes please! I just had to sign in today to use these. 🤣

**Comment by ojeytonwilliams** [comment]
There are a11y concerns for this which @bbsmooth brought up (starting here: https://github.com/freeCodeCamp/freeCodeCamp/pull/45530#issuecomment-1101796368).

The gist was that, because our shortcuts are single, unmodified characters, they can easily be triggered accidentally. The solution was to make the shortcuts toggleable and disabled by default. I think we should preserve that solution for signed-out users.

A keyboard shortcut toggle in the menu could work. The state could go into local storage and be superseded by the value in the user-record if they sign in.

**Comment by lasjorg** [comment]
I would be OK with a menu option for it. Also, for some reason I thought they were enabled by default for signed-in users, but maybe they aren't?

Speaking of which, I was going to open another issue about the theme settings for signed-out users. It seems a little odd that you can't set the theme (dark/light) unless you are signed in. I don't think that is a common UX pattern (kind of reminds me of "pay for dark mode").

**Comment by lasjorg** [comment]
That makes sense.

Just so I'm clear, what is the procedure for testing a new user in dev. How do you seed the DB and how do you use the blank user? If I just seed the DB normally, and sign-out and then try to "sign-up" (i.e. sign-in) it just signs me back in to the developer user. 

Do you just not seed the DB? I'm confused about the `blankUser` in `seed-demo-user.js`, how and what is that used for?

**Comment by ojeytonwilliams** [comment]
It's a pain when you're using the current api, since it does not create a new user when you sign-in in development. You have to go in via Auth0 (i.e. setup all the Auth0 keys in .env and set `LOCAL_MOCK_AUTH=false`).

Things are a little better in the new api: if you drop the db and sign-in, it will create a fresh user for you. It still has the email address, "foo@bar.com", but it creates all the other properties (like `keyboardShortcut`) the same way in development as in production.

As for `blankUser`, it's just there for some e2e tests. You can't log into it.

**Comment by ojeytonwilliams** [comment]
> Do you just not seed the DB? 

No, because the seed script makes a mongo query directly. It's not necessarily exactly the same query that the api makes when it creates a user. This is a bit confusing, but it does let us setup users how we want (mostly relevant in testing).

It might be better to always use the `signin` route if we want to create a user, but I'm not sure.

**Comment by ojeytonwilliams** [comment]
@Sembauke I think this is a reasonable feature request. As long as it's implemented in an a11y way, it should be useful.

**Comment by ahmaxed** [comment]
The implementation and the UI is not that straight forward. Since users can sign in to use the shortcut, having this feature might not be necessary.
